macosx/CreatorWindowController.mm

```diff
@@ -360,7 +360,20 @@ - (IBAction)cancelCreateProgress:(id)sender
 
 - (IBAction)incrementOrDecrementPieceSize:(id)sender
 {
-    auto const piece_size = static_cast<uint32_t>(pow(2.0, [sender intValue]));
+    auto power = [(NSStepper*)sender intValue];
```

Comment:
Can't we just set min/max value on NSStepper itself?
https://developer.apple.com/documentation/appkit/nsstepper